gcp-observability/src/main/java/io/grpc/gcp/observability/GlobalLoggingTags.java

```diff
@@ -62,10 +59,6 @@ Map<String, String> getLocationTags() {
     return locationTags;
   }
 
-  Map<String, String> getCustomTags() {
-    return customTags;
-  }
-
   @VisibleForTesting
   static void populateFromMetadataServer(ImmutableMap.Builder<String, String> customTags) {
```

Comment:
Suggestion: This file is being used only to take location tags for logging.  How about we rename instances of `customTags` to `locationTags` everywhere else in the file?

Reply:
done